edx_notifications/server/web/static/edx_notifications/js/models/counter_icon_model.js

```diff
@@ -0,0 +1,11 @@
+var CounterIconModel = Backbone.Model.extend({
```

Comment:
Thanks, looking at search_result.js, I'm not sure about the syntax. I'll talk to @martynjames or you tomorrow when we are back in the office.